frontend/src/components/dashboard/geo_location/GroupedTable.vue

```diff
@@ -75,8 +75,10 @@
             geoLocationItems() {
                 return this.items.map((item) => {
                     let newItemData = {
-                        avg_session_time: moment.unix(item.avg_session_time).format("HH:mm:ss"),
-                        bounce_rate: Math.round(Number(item.bounce_rate)*100)
+                        avg_session_time: moment.utc(
```

Comment:
this formatting won't work properly if the average duration will be more than 24 hours